### PR TITLE
Updates to match latest EVM market maker API changes

### PR DESCRIPTION
**Pull request by mitch-dflow**

### CODE REVIEW

File: spec/EvmLegacy.yml [hunk 1]
```diff
@@ -81,11 +81,61 @@ paths:
...
+        - types
+        - domain
+        - primaryType
+
     IndicativeQuoteGetResponse:
       type: object
       properties:
         fillPrice:
-          description: Token units sent to retail trader per token received from retail trader. Must be greater than or equal to the minFillPrice.
+          description: oken units sent to retail trader per token received from retail trader. Must be greater than or equal to the minFillPrice.
```
Comment (jeff-delonge) [inline spec/EvmLegacy.yml:138]: Fix typo

File: spec/EvmLegacy.yml
Comment (jeff-delonge) [inline spec/EvmLegacy.yml:1]: Rename `EvmStandard.yml`, rename "EVM Standard Swaps"

File: spec/EvmLegacy.yml [hunk 2]
```diff
@@ -190,23 +270,19 @@ components:
...
-          pattern: '^[0-9a-fA-F]+$'
-        salt:
-          description: salt used to construct the 0x RFQ order. Formatted as a hex string.
-          type: string
-          pattern: '^[0-9a-fA-F]+$'
+          description: expiry used to construct the DFlow Swap order
+          type: integer
+          minimum: 0
+          maximum: 1099511627775  # Equivalent to 0xffffffffff in hexadecimal
         rfqOrderSignature:
```
Comment (jeff-delonge) [inline spec/EvmLegacy.yml:284]: Fix field name

**Review by jeff-delonge — CHANGES_REQUESTED** [review]

File: spec/EvmLegacy.yml [hunk 3]
```diff
@@ -95,22 +145,38 @@ components:
...
+          $ref: '#/components/schemas/SchemaAddress'
+        takerToken:
+          description: takerToken to use when constructing the DFlow Swap order. Not specified if the retail trader is sending the unwrapped variant of the native token.
+          $ref: '#/components/schemas/SchemaAddress'
+        makerToken:
+          description: makerToken to use when constructing the DFlow Swap order. Not specified if the retail trader is receiving the unwrapped variant of the native token.
+          $ref: '#/components/schemas/SchemaAddress'
         sendQty:
-          description: Send quantity specified as a scaled integer
+          description: Send quantity specified as a scaled integer. This is the `takerAmount` to use when constructing the DFlow Swap order.
```
Comment (jeff-delonge) [inline spec/EvmLegacy.yml:160]: The descriptions aren't using backticks around field names consistently. We should be consistent and either use backticks in all or none of the descriptions

Comment (mitch-dflow) [reply spec/EvmLegacy.yml:160]: the issue I found is that you can't use a backtick to start a string field so if it was at the start it was causing problems. I'll remove them all

**Review by jeff-delonge — APPROVED** [review]

**Pull request merged**